lib/main.dart

```diff
@@ -17,7 +18,7 @@ void main() {
       theme: ThemeData(
         primarySwatch: Colors.deepPurple,
       ),
-      home: const MyApp()));
+      home: SplashScreen()));
```

Comment:
This should be from native platform, We should write code for splash otherwise it would show a white screen in IOS devices, and sometimes apple reject the apps.